### PR TITLE
Enable publishing of features/readonly-ref artefacts.

### PR DESCRIPTION
**Pull request by VSadov**

### CODE REVIEW

File: build/config/PublishData.json [hunk 1]
```diff
@@ -13,6 +13,13 @@
             "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
             "channels": [ "publish-test" ]
+        },
+        "features/readonly-ref": {
+            "nugetKind": "PerBuildPreRelease",
+            "version": "2.6.*",
+            "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
+            "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
+            "channels": [ "dev15.5", "dev15.5p1" ]
```
Comment (jaredpar) [inline build/config/PublishData.json]: Need to update channels to the following

``` json
"channels": [  "ref-readonly" ]
```

File: build/config/PublishData.json [hunk 2]
```diff
@@ -13,6 +13,13 @@
             "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
             "channels": [ "publish-test" ]
+        },
+        "features/readonly-ref": {
+            "nugetKind": "PerBuildPreRelease",
+            "version": "2.6.*",
+            "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
+            "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
```
Comment (jaredpar) [inline build/config/PublishData.json]: Leave the vsix empty for now.

``` json
"vsix": [ ]
```

Comment (VSadov) [reply build/config/PublishData.json]: will it publish the vsix without that?

**Comment by VSadov** [comment]
The process cannot access the file

**Comment by VSadov** [comment]
test windows_release_unit64_prtest please

File: build/config/PublishData.json (hunk 2 above)
Comment (VSadov) [reply build/config/PublishData.json]: I've put the "vsix" back. We would like to see vsix published as well for this branch.

**Comment by VSadov** [comment]
test windows_debug_unit32_prtest please

**Comment by VSadov** [comment]
test windows_debug_vs-integration_prtest please

**Pull request merged**